### PR TITLE
[4965][ADD] stock_move_value

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[4965](https://www.quartile.co/web#id=4965&cids=3&model=project.task&view_type=form)

### CODE REVIEW

File: stock_move_value/views/stock_move_views.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        </field>
+    </record>
+    <record id="view_move_tree" model="ir.ui.view">
+        <field name="name">stock.move.tree</field>
+        <field name="model">stock.move</field>
+        <field name="inherit_id" ref="stock.view_move_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='company_id']" position="before">
+                <field
+                    name="move_value"
```
Comment (yostashiro) [inline stock_move_value/views/stock_move_views.xml:35]:
```suggestion
                    name="move_value"
                    sum="Total Value"
```

File: stock_move_value/views/stock_move_views.xml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                    groups="account.group_account_invoice"
+                    optional="show"
+                />
+                <field
+                    name="move_origin_value"
```
Comment (yostashiro) [inline stock_move_value/views/stock_move_views.xml:41]:
```suggestion
                    name="move_origin_value"
                    sum="Total Origin Value"
```

File: stock_move_value/views/stock_move_views.xml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                    groups="account.group_account_invoice"
+                    optional="show"
+                />
+                <field
+                    name="value_discrepancy"
```
Comment (yostashiro) [inline stock_move_value/views/stock_move_views.xml:47]:
```suggestion
                    name="value_discrepancy"
                    sum="Total Discrepancy"
```

**Review by yostashiro — APPROVED** [review]
Just some minor points to update as time allows.

**Review by yostashiro — APPROVED** [review]

**Review by kanda999 — APPROVED** [review]
Functional review.

**Pull request merged**